packages/multichain-network-controller/src/constants.ts

```diff
@@ -0,0 +1,55 @@
+import { BtcScope, SolScope } from '@metamask/keyring-api';
+import { NetworkStatus } from '@metamask/network-controller';
+
+import type {
+  MultichainNetworkConfiguration,
+  MultichainNetworkMetadata,
+} from './MultichainNetworkController';
+
+export const btcNativeAsset = `${BtcScope.Mainnet}/slip44:0`;
+export const solNativeAsset = `${SolScope.Mainnet}/token:EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v`;
+
+export const multichainNetworkConfigurations: Record<
```

Comment:
What are your thoughts on using `SCREAMING_SNAKE_CASE` and also using the word "available" (to communicate that these are things that you _could_ initialize the state with, but you don't have to use all of them).

``` suggestion
export const AVAILABLE_MULTICHAIN_NETWORK_CONFIGURATIONS: Record<
```